packages/rhi-webgl/src/GLRenderTarget.ts

```
@@ -11,22 +11,209 @@
+   * @param attachment - Framebuffer attachment point
+   * @returns Created and configured WebGL render buffer
+   */
+  static createRenderBuffer(
+    gl: WebGLRenderingContext & WebGL2RenderingContext,
+    target: RenderTarget,
+    internalFormat: GLenum,
+    attachment: GLenum
+  ): WebGLRenderbuffer {
+    const renderBuffer = gl.createRenderbuffer();
+    const { width, height, antiAliasing } = target;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)